src/app_modes/topic_managment.rs

```diff
@@ -268,6 +276,16 @@ impl TopicManager {
                     topic: topic[0].clone(),
                     rotation: 0,
                 }),
+                "geometry_msgs/PolygonStamped" => {
```

Comment:
do I understand correctly that every time we save the config we will get a different color?

Maybe we should reuse the color of the previous config for the topic that were already added from the beginning?

Reply:
this is the same for all other topics. So lets keep it for this pr